### DISCUSSION TITLE
[IntelliSense] Doesn't work with TypeScript properties when using backticks

### DISCUSSION THREAD

**Discussion by radarsu** [tailwindlabs/tailwindcss#12680] [Ideas]
Autocomplete doesn't work when in TypeScript file using backtick `(``)` instead of classic quotes.

**Comment by wongjn** [comment]
You could try configuring the [`tailwindCSS.experimental.classRegex` setting](https://github.com/tailwindlabs/tailwindcss/issues/7553#issuecomment-735915659).

**Comment by radarsu** [comment]
Thanks, this rule worked for me:
```json
"tailwindCSS.experimental.classRegex": ["class ?= ?`(.*)`"],
```
